src/app/pages/apps/components/available-apps/available-apps.component.html

```diff
@@ -4,10 +4,22 @@
   </ix-page-title-header>
 </ng-template>
 
-<ix-available-apps-header (filters)="changeFilters($event)"></ix-available-apps-header>
+<ix-available-apps-header
+  (filters)="changeFilters($event)"
+  (search)="changeSearchQuery($event)"
+></ix-available-apps-header>
 
-<div *ngIf="filters.search" class="apps-title"> {{ 'Search Results for «{query}»' | translate: { query: filters.search } }}</div>
-<div *ngIf="apps.length" class="available-apps-list">
+<div *ngIf="searchQuery" class="apps-title"> {{ 'Search Results for «{query}»' | translate: { query: searchQuery } }}</div>
```

Comment:
![Kapture 2023-03-24 at 12 59 27](https://user-images.githubusercontent.com/351613/227437294-3c5f0288-d850-4419-b857-b5de846e5d7c.gif)

Let's use styles like in category titles.